### ISSUE TITLE
v0.13.0 build failure due to bitcoind incompatibility

### ISSUE THREAD

**Issue by rajarshimaitra** [RCasatta/electrsd#27]
Due to recent changes made in https://github.com/RCasatta/bitcoind/pulls?q=is%3Apr+is%3Aclosed the current `master` build is failing.

```
$ cargo build
   Compiling bitcoind v0.21.0
error: cannot find derive macro `Display` in this scope
  --> /home/raj/.cargo/registry/src/github.com-1ecc6299db9ec823/bitcoind-0.21.0/src/lib.rs:71:17
   |
71 | #[derive(Debug, Display)]
   |                 ^^^^^^^

error: could not compile `bitcoind` due to previous error
```
Because master still points to `bitcoind v0.21.0` while having its `Error` `std::error::Error`. 

Also because the `v0.13.0` is not tagged yet, I think the `electrsd v0.13.0` in cargo is actually building the current `master` so I am seeing the failure in downstream too.

Simple fix for people with the same issue : Use electrsd `v0.12.0` in your Cargo.toml until `v0.13.0` is tagged.

**Comment by RCasatta** [comment]
I am not sure to follow, both bitcoind and electrsd with std::error::Error impl aren't published on crates yet

**Comment by RCasatta** [comment]
I would have needed to publish bitcoind and update the dependency in electrsd to make electrsd build, not sure why the CI worked

I have to check as soon I get a computer (on the phone right now)

**Comment by rajarshimaitra** [comment]
>I am not sure to follow, both bitcoind and electrsd with std::error::Error impl aren't published on crates yet

The gist of the issue as far as I understand is 
 - current master is trying to set the Error `std::Error` while `bitcoind::Error` is still not `Display` in `v0.21.0`.
 - building `electrsd v0.13.0` is also giving the error because `v0.13.0` is not yet tagged in github? So its building the current master instead and hitting the same issue.
 ```
$ git log --oneline
2a8a297 (HEAD -> master, origin/pr/26, origin/master, origin/HEAD, myown/error-fix, error-fix) Implement std::error::Error for crate::Error
0d13ac5 bump version to 0.13.0
8bb9034 bump bitcoind dep to 0.21
38b1add Update Readme.md
4563162 (tag: 0.12.0, origin/pr/24, origin/bitcoind020) bump version to 0.12.0
a82282b bump bitcoind dep to 0.20.0
4679e56 (origin/pr/22, origin/latest_electrs) fix electrs_0_9_1 p2p requirements
cc3e9cf add electrs_0_9_1 feature
82a3d14 make lower case work in both mac11 and ubuntu 20
bc3f79c updated tag to build latest electrs
c141e14 (origin/pr/23) Remove zip dependency time feature
c6a281a (tag: 0.11.0) bump version to 0.11.0
```

**Comment by RCasatta** [comment]
I can't reproduce...

However, I think having `std::error::Error` is a reason to release a new version, just released `bitcoind 0.24.0` and `electrsd 0.14.0`

**Comment by rajarshimaitra** [comment]
Thanks.. Verified that the issues goes away in v0.14.0.
